sig-auth/README.md

```diff
@@ -37,6 +37,39 @@ The Chairs of the SIG run operations and processes governing the SIG.
 * [Mailing list](https://groups.google.com/forum/#!forum/kubernetes-sig-auth)
 * [Open Community Issues/PRs](https://github.com/kubernetes/community/labels/sig%2Fauth)
 
+## Subprojects
+
+The following subprojects are owned by sig-auth:
+- **audit-logging**
+  - Owners:
+    - Tim Allclair
+- **authenticators**
+  - Owners:
+    - Mike Danese
+    - Jordan Liggitt
+- **authorizers**
+  - Owners:
+    - Jordan Liggitt
+    - Mike Danese
+    - David Eads
+- **cross-cutting-documentation**
+  - Owners:
+    - NEEDS OWNER
+- **encryption-at-rest**
+  - Owners:
+    - NEEDS OWNER
+- **node-identity-and-isolation**
+  - Owners:
+    - Mike Danese
+    - Jordan Liggitt
+- **policy-management**
```

Comment:
Worth calling out the the policy working group (now merged with SAFE)?

Reply:
(meetings field?)

Reply:
Is this project supposed to represent a topic, or actual maintenance and development on the below subtrees? How involved is SAFE in the later?